README.md

```diff
@@ -9,6 +9,10 @@ This repo houses the new version of the Deakin Detonator Toolkit application bui
 `src/` contains the source code for the UI.
 `src-tauri` contains the source code and configuration for the Tauri application.
 
+## What is the Deakin Detonator Toolkit? 
+
+In its simplest definition, Deakin Detonator Toolkit is a penetration testing toolkit. The toolkit allows you to use a variety of tools, without needing the "know-how" of each command.
```

Comment:
it's

Reply:
Hi Bradley, 

I reckon it should be _its_ because _it's_ is an abbreviation for "_it is_". _Its_ is possessive. 

If it looks strange, maybe we can try a different wording altogether? For example, _The Deakin Detonator Toolkit's main role is as a toolkit for penetration testing._

Reply:
yep. My bad. Cheers